testdata/mod/rsc.io_sampler_v1.2.1.txt

```diff
@@ -0,0 +1,134 @@
+generated by ./addmod.bash rsc.io/sampler@v1.2.1
```

Comment:
Where does `addmod.bash` live?

Is it worth adding a `.gitattributes` for such generated files so they don't appear expanded in the GitHub PR diff view?

Reply:
I just copied the file from $GOROOT. Interestingly, the comment is out of date even there, as addmod.bash is now src/cmd/go/testdata/addmod.go. The go-internal project now exports the equivalent txtar-addmod command.

I'm not sure about adding the .gitattributes file - the Go project doesn't have one, and these files are designed for human consumption, so I think it's good for them to appear in reviews.